linux/gnome/src/main/java/com/github/weisj/darkmode/platform/linux/gnome/GnomeThemeMonitorService.java

```diff
@@ -30,17 +30,21 @@
 
     @Override
     public boolean isDarkThemeEnabled() {
-        // TODO: Stop guessing and check the settings when available (like what is mentioned in the GtkVariants class)
```

Comment:
The wording in `GtkVariants` should be updated as well.

Reply:
Completed. See 7c6fc0c97c971afc4cb6e24cb04ce54fd918011c.